packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -764,6 +764,77 @@ void main() {
     expect(item5material.color, Colors.transparent); // the previous item should not be highlighted.
   }, variant: TargetPlatformVariant.desktop());
 
+  // Regression test for https://github.com/flutter/flutter/issues/147253.
+    testWidgets('Down key and up key can navigate on desktop platforms when a label text contains another label text',
+    (WidgetTester tester) async {
+    final ThemeData themeData = ThemeData();
+    await tester.pumpWidget( MaterialApp(
+      theme: themeData,
+      home: const Scaffold(
+        body: DropdownMenu<int>(
+          dropdownMenuEntries: <DropdownMenuEntry<int>>[
+            DropdownMenuEntry<int>(
+              value: 0,
+              label: 'ABC'
+            ),
+            DropdownMenuEntry<int>(
+              value: 1,
+              label: 'AB'
+            ),
+            DropdownMenuEntry<int>(
+              value: 2,
+              label: 'ABCD'
+            ),
+          ],
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byType(DropdownMenu<int>));
+    await tester.pump();
+
+    final Finder button0Material = find.descendant(
+      of: find.widgetWithText(MenuItemButton, 'ABC').last,
+      matching: find.byType(Material),
+    );
+    final Finder button1Material = find.descendant(
+      of: find.widgetWithText(MenuItemButton, 'AB').last,
+      matching: find.byType(Material),
+    );
+    final Finder button2Material = find.descendant(
+      of: find.widgetWithText(MenuItemButton, 'ABCD').last,
+      matching: find.byType(Material),
+    );
+
+    // Press down key three time, the highlight should move to the next item each time.
```

Comment:
nit: three times

Reply:
Thanks for the suggestion. I've fixed the above issues.